signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/OrganizerSignupMBean.java

```diff
@@ -1241,7 +1241,12 @@ public void setUserInputEidOrEmail(String value) {
 	 * @return a list of SignupSite objects.
 	 */
 	public List<SignupSite> getPublishedSignupSites() {
-		return getMeetingWrapper().getMeeting().getSignupSites();
+		try {
+			return getMeetingWrapper().getMeeting().getSignupSites();
+		} catch (NullPointerException npe) {

```

Comment:
Why should this particular bit of code have a catch for a NPE. What's the case that it's trying to handle and wouldn't it be better to explicitly code for it with a check?